checkout/interfaces/graphql/mutationresolver.go

```diff
@@ -100,15 +100,6 @@ func (r *CommerceCheckoutMutationResolver) CommerceCheckoutStartPlaceOrder(ctx c
 	}
 	startPlaceOrderCommand := placeorder.StartPlaceOrderCommand{Cart: *cart, ReturnURL: returnURL}
 	pctx, err := r.placeorderHandler.StartPlaceOrder(ctx, startPlaceOrderCommand)
-	if err == placeorder.ErrAnotherPlaceOrderProcessRunning {
```

Comment:
is this error not thrown anymore? I do not see any changes that remove the throwing of this one. If it is still thrown it must be ignored specifically I guess?